components/textFields/AdvancedOpenTextField.js

```diff
@@ -30,66 +32,52 @@ const AdvancedOpenTextField = (props) => {
     // Populate if value already exists in redux
     if(!value) {
         if(existingData != null) {
-            if(existingData[currId] != null && !value) {
+            if(existingData[currId] != null && existingData[currId] != '' && !value) {
                 setValue(existingData[currId]);
-                setButtonAppearance('filled');
-                setAdvancedButtonAppearance("filled")
             }
         }
     }
 
-    const submitField = () => {
-      //submits data to reducer and is shown as complete
-        if(!value) {
-            return;
-        }
-        submitFunction({id, question: currId, selection: value})
-        switch(currId) {
-          case "Bw7d2KTr": // VIN question id
-            if (!vinValidator.validate(value)) {
-              setInvalidVin(true);
-            } else {
-              setInvalidVin(false);
-            }
-            break;
-          default:
-            break;
-        }
-        ErrorMsg();
-        setButtonAppearance('filled');
-    }
-
     const onTextChange = (text) => {
```

Comment:
This function appears in every file. Based on my first glance all of them seem to be the same. Is there any reason why the function has to be repeated in each file? Could the function be added to the `helperFuctions.js` and called from there?

Reply:
Yeah, it's because of an issue I was having with imports. There's potential to port this over into the helperFunctions.js section, but I was having difficulty componentizing it so I left it here for the time being. Happy to look at it together.